data.json

```diff
@@ -270,5 +270,16 @@
         "persons": [
             "Camille Fournier"
         ]
+    },
+	"https://www.youtube.com/playlist?list=PLrhzvIcii6GNjpARdnO4ueTUAVR9eMBpc": {
+        "name": "Design Patterns in Object Oriented Programming",
+        "tags": [
+            "talk",
```

Comment:
I'm not entirely sure if talk matches, all other talks are some kind of presentation for conferences. Maybe `course` would fit better?